Cython/Utility/ModuleSetupCode.c

```diff
@@ -237,7 +237,7 @@
   #undef CYTHON_USE_MODULE_STATE
   #define CYTHON_USE_MODULE_STATE 1
   #ifndef CYTHON_USE_TP_FINALIZE
-    #define CYTHON_USE_TP_FINALIZE 1
+    #define CYTHON_USE_TP_FINALIZE 0
```

Comment:
Hmm … does this not work with `PyType_GetSlot()`?

Reply:
It's `PyObject_CallFinalizerFromDealloc`. I don't think we can easily replace that using the tools in the limited API, but I'll add an explanatory comment here so it's at least clear what's missing.